server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -83,16 +72,64 @@ export class ChatController implements ChatHandlers {
             )
         }
 
-        const result = await this.#processAssistantResponse(response)
+        const result = await this.#processAssistantResponse(response, params.partialResultToken)
```

Comment:
Is this deliberately outside the `try/catch`? Is there no way that parsing chat events from the server response can fail? Or if they can, would we need a specific exception?